src/DocumentFormat.OpenXml/OpenXmlPartReader.cs

```diff
@@ -451,6 +451,12 @@ private bool MoveToNextElement()
                 case ElementState.MiscNode:
                     // cursor is end element, pop stack
                     _elementStack.Pop();
+                    if (_elementStack.Count == 0)
```

Comment:
This looks like it was a bug that just wasn't caught before :)

Reply:
Before the whitespace change this was handeled by 
```csharp
if (_xmlReader.EOF || !_xmlReader.Read())
{
  _elementState = ElementState.EOF;
  return false;
}
```
further down in the function. With the whitespace change the stack can be empty but the reader is not at the end of file as it still hat whitespace to read